packages/ui-components/src/Button/Button.tsx

```diff
@@ -8,8 +8,7 @@ import styles from "./Button.module.scss";
 export type ButtonIntent =
   | "primary-success"
   | "primary-danger"
-  | "secondary"
-  | "tertiary";
```

Comment:
What happened to tertiary intent? I still see it in Figma.

Reply:
I see, in console it's called a flat button so I can keep the tertiary in ui-components and later update in console (probably when the button component is edited to be able to be used in console).